content/en/software/using-medley/cl-using.md

```diff
@@ -2,27 +2,41 @@
 title: Interlisp Basics for Common Lisp users
 url: /software/using-medley/cl-using/
 type: docs
-weight: 20
+weight: 30
 aliases:
  - 
 ---
-## Interlisp Basics for Common Lisp users
+# Interlisp Basics for Common Lisp users
```

Comment:
Remove this line, the title will be generated from the `yaml` title block above, line #2.